CI/before_script.msvc.sh

```diff
@@ -415,17 +413,23 @@ fi
 	if [ -z $APPVEYOR ]; then
 		cd $DEPS_INSTALL
 
+        # Boost's installer is still based on ms-dos API that doesn't support larger than 260 char path names
+        # We work around this by installing to root of the current working drive and then move it to our deps
+        
 		BOOST_SDK="$(real_pwd)/Boost"
+		CWD_DRIVE_ROOT=`powershell -command '(get-location).Drive.Root'`  # get the current working drive's root
 
 		if [ -d Boost ] && grep "BOOST_VERSION 106100" Boost/boost/version.hpp > /dev/null; then
```

Comment:
You'll need to update the value in this check, otherwise it's going to replace the boost copy every time the script is run, even if it's already current.